### ISSUE TITLE
your ckpt?

### ISSUE THREAD

**Issue by Keysmis** [namedysx/CRAFT-tensorflow#1]
could you provided your model file? the CRAFT model。。
thanks~ best wishes

**Comment by namedysx** [comment]
> could you provided your model file? the CRAFT model。。
> thanks~ best wishes

https://pan.baidu.com/s/1l--zW-Qb_0m7q-phoaFNMg
When I have time I'll upload it to Google drive.

**Comment by zzks** [comment]
Thank you for the tensorflow version craft !
Error shows when load this ckpt:
NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint
         [[Node: save/RestoreV2_79 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_79/tensor_names, save/RestoreV2_79/shape_and_slices)]]
         [[Node: save/RestoreV2_103/_217 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_464_save/RestoreV2_103", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

how to fix it? thank you!

**Comment by mkara44** [comment]
> > could you provided your model file? the CRAFT model。。
> > thanks~ best wishes
> 
> https://pan.baidu.com/s/1l--zW-Qb_0m7q-phoaFNMg
> When I have time I'll upload it to Google drive.

Link doesn't work :(

**Comment by Gaurav14cs17** [comment]
plz upload it to google drive

**Comment by namedysx** [comment]
> Thank you for the tensorflow version craft !
> Error shows when load this ckpt:
> NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint
> [[Node: save/RestoreV2_79 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_79/tensor_names, save/RestoreV2_79/shape_and_slices)]]
> [[Node: save/RestoreV2_103/_217 = _Recvclient_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_464_save/RestoreV2_103", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]]
> 
> how to fix it? thank you!

Sorry for my late response. 
try to check vgg16.ckpt path

**Comment by namedysx** [comment]
> > > could you provided your model file? the CRAFT model。。
> > > thanks~ best wishes
> > 
> > 
> > https://pan.baidu.com/s/1l--zW-Qb_0m7q-phoaFNMg
> > When I have time I'll upload it to Google drive.
> 
> Link doesn't work :(


> plz upload it to google drive


https://drive.google.com/file/d/1-AL-T3-WucUZsWMUHqyWovuXEvqYZCi9/view?usp=drivesdk

**Comment by zzks** [comment]
> > Thank you for the tensorflow version craft !
> > Error shows when load this ckpt:
> > NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint
> > [[Node: save/RestoreV2_79 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_79/tensor_names, save/RestoreV2_79/shape_and_slices)]]
> > [[Node: save/RestoreV2_103/_217 = _Recvclient_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_464_save/RestoreV2_103", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]]
> > how to fix it? thank you!
> 
> Sorry for my late response.
> try to check vgg16.ckpt path

Thanks for your relpy!
I try to load your ckpt and do some inference.
this error shows when I try to restore your ckpt (CRAFT_26200.ckpt):
restore.restore(sess, "/media/user1/test/CRAFT_26200.ckpt")
please give me some advice, Thank you!

**Comment by namedysx** [comment]
> > > Thank you for the tensorflow version craft !
> > > Error shows when load this ckpt:
> > > NotFoundError (see above for traceback): Key vgg_16/Repeat/convolution_1/BatchNorm/beta not found in checkpoint
> > > [[Node: save/RestoreV2_79 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2_79/tensor_names, save/RestoreV2_79/shape_and_slices)]]
> > > [[Node: save/RestoreV2_103/_217 = _Recvclient_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_464_save/RestoreV2_103", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]]
> > > how to fix it? thank you!
> > 
> > 
> > Sorry for my late response.
> > try to check vgg16.ckpt path
> 
> Thanks for your relpy!
> I try to load your ckpt and do some inference.
> this error shows when I try to restore your ckpt (CRAFT_26200.ckpt):
> restore.restore(sess, "/media/user1/test/CRAFT_26200.ckpt")
> please give me some advice, Thank you!

Could u train only use vgg16.ckpt?
I didn't have that problem.


you can try to modify that.

--     sess.run(tf.global_variables_initializer())
        if train:
++      sess.run(tf.global_variables_initializer())
            print('-----load vgg-----')
            restorer.restore(sess, '/home/user4/ysx/CRAFT/model/vgg16.ckpt')
            print('-----load vgg complete-----')
            print('-----training-----')
        else:
            print('-----load ckpt-----')
            restorer.restore(sess, '/home/user4/ysx/sigma3_demo/CRAFT_262000.ckpt')
            print('-----load ckpt complete')
            print('-----training------')

maybe it could work.

**Comment by namedysx** [comment]
#2 
or you can try this
